server/src/services/IrmaService.ts

```diff
@@ -40,6 +45,21 @@ export default class {
         return await this.irmaBackend.startSession(jwt);
     };
 
+    public requestDisclosureSession = async (
+        credentialsToRequest: CredentialSet
+    ): Promise<{ sessionPtr: string; token: string }> => {
+        const request = this.createDisclosureRequest(credentialsToRequest);
+        return await this.requestSession(request);
+    };
+
+    public requestSignatureSession = async (
```

Comment:
Ik kan het uit de diff in github niet goed opmaken, maar aan requestDisclosureSession heb je nu niets veranderd toch?

Reply:
nee, behalve dat ik een deel ervan naar een generieke functie (requestSession) heb overgezet